### PR TITLE
fix: update with latest dependency changes

### LINKED ISSUE #5: fix(DMT): make app work again

### PR DESCRIPTION
**Pull request by soofstad**
## What does this pull request change?
- Update the app to work with latest changes in development framework dependencies
- Update NPM dependencies
- Update pre-commit hooks
- Update readme (recipe lookup)

## Why is this pull request needed?
- DMT should work with latest DM components

## Issues related to this change
closes #5

### CODE REVIEW

File: docker-compose.override.yml [hunk 1]
```diff
@@ -22,13 +22,13 @@ services:
...
+#  db-ui:
+#    image: mongo-express:1.0.0-alpha
+#    restart: unless-stopped
+#    ports:
+#      - "8082:8081"
+#    environment:
+#      ME_CONFIG_MONGODB_SERVER: db
+#      ME_CONFIG_MONGODB_ADMINUSERNAME: maf
+#      ME_CONFIG_MONGODB_ADMINPASSWORD: maf
+#      ME_CONFIG_MONGODB_ENABLE_ADMIN: "true"
```
Comment (KristianKjerstad) [inline docker-compose.override.yml]: is this commented out by mistake?

File: README.md [hunk 2]
```diff
@@ -26,26 +26,22 @@ Let the web app know of which DMSS to use by setting the `DMSS_URL` environment
 ### Starting
 
 ```shell
-docker-compose up
+docker-compose pull
+docker-compose up --build
```
Comment (KristianKjerstad) [inline README.md:30]: just a note: this command fails when connected to internal Equinor network, but works on "statoil aproved"

Would be nice to add this to the readme, but a proper fix might not be necessary 

image

**Review by KristianKjerstad — APPROVED** [review]
good!

tested locally, and the app seems to start.

There are still some bugs (like the actions "new entity", "new folder", etc in the context menu)
but I guess we can fix them later

File: docker-compose.override.yml (hunk 1 above)
Comment (soofstad) [reply docker-compose.override.yml]: yepp, fixed

**Pull request merged**